WordPress/Classes/ViewRelated/Stats/Extensions/Double+Stats.swift

```diff
@@ -13,6 +19,34 @@ extension Double {
         }
     }
 
+    private var decimalFormatter: NumberFormatter {
+         get {
+             let formatter = Double._decimalFormatter
+             // Show at least one digit after the decimal
+             formatter.minimumFractionDigits = 1
+             return formatter
```

Comment:
Nitpick: Just noting that these lines are indented differently compared to the contents of `numberFormatter` above. 😄